app/build.gradle

```diff
@@ -37,19 +37,26 @@ dependencies {
     implementation "com.android.support:recyclerview-v7:$supportLibraryVersion"
     implementation "com.android.support.constraint:constraint-layout:$constrainLayoutVersion"
     implementation 'com.android.support:multidex:1.0.3'
+    implementation 'com.android.volley:volley:1.1.1'
 
     implementation "android.arch.lifecycle:extensions:$lifecycle_version"
     kapt "android.arch.lifecycle:compiler:$lifecycle_version"
 
     implementation "com.google.android:flexbox:$flexboxVersion"
 
+    implementation ('com.squareup.picasso:picasso:2.71828') {
```

Comment:
Curious - this is to exclude duplicate dependencies?

Reply:
Yeah, picasso uses duplicate dependencies internally so i need to exclude them to build properly